src/lib/components/AnnouncementBanner/AnnouncementBanner.svelte

```diff
@@ -0,0 +1,33 @@
+<script lang="ts">
+  import "./AnnouncementBanner.scss";
+  import Link from "$lib/components/Link";
+  import Icon from "$lib/components/Icon";
+  import { url as lightbulbIcon } from "$icons/lightbulb";
+  import { url as arrowIcon } from "$icons/arrow_forward";
+  import classNames from "$lib/util/classNames";
+
+  export let headerText = "Here's a site-wide announcement!";
+  export let linkText = "Read more here";
+  export let href = "/";
+  let classes = classNames("usa-banner", `ldaf-banner--announcement`);
+</script>
+
+<section id="top" class={classes} aria-label="Official website of the Louisiana state government">
+  <header class="usa-banner__header">
+    <div class="usa-banner__inner">
+      <div class="grid-col-auto">
+        <img
+          class="usa-banner__header-flag ldaf-logo"
```

Comment:
The `ldaf-logo` class probably isn't appropriate here, right?

Reply:
yeah, that was an accidental carryover from splitting the 2 banners up. Renaming that to be more accurate.